Python/Tests/Core/WorkspaceInterpreterFactoryTests.cs

```diff
@@ -159,15 +159,74 @@ public class WorkspaceInterpreterFactoryTests {
             }
         }
 
-        private static IEnumerable<InterpreterConfiguration> TestTriggerDiscovery(IVsFolderWorkspaceService workspaceService, Action triggerDiscovery) {
+        [TestMethod, Priority(0)]
+        public void WatchWorkspaceVirtualEnvRenamed() {
+            const string ENV_NAME = "env";
+            var workspaceService = CreateEnvAndGetService(ENV_NAME);
+
+            string envPath = Path.Combine(workspaceService.CurrentWorkspace.Location, ENV_NAME);
+            var configs = TestTriggerDiscovery
+                (workspaceService, () => Directory.Move(envPath, string.Concat(envPath, "1"))).ToArray();
+
+            Assert.AreEqual(1, configs.Length);
+            Assert.IsTrue(PathUtils.IsSamePath(
+                Path.Combine(string.Concat(envPath, "1"), "scripts", "python.exe"),
+                configs[0].InterpreterPath));
+            Assert.AreEqual("Workspace|Workspace|env1", configs[0].Id);
+
+            Directory.Delete(workspaceService.CurrentWorkspace.Location, true);
```

Comment:
don't delete it. it's in temp location, so there's no need to clean it up.

if you were modifying something in a "real location" on the machine, that might affect further tests for example, then that's a case where you would want to clean up.

Reply:
Removed